types/react-dom/experimental.d.ts

```diff
@@ -49,4 +49,25 @@ declare module "react" {
 
     // eslint-disable-next-line @typescript-eslint/no-empty-interface
     interface GestureProvider extends AnimationTimeline {}
+
+    // @enableFragmentRefs
+    interface FragmentInstance {
+        blur: () => void;
+        focus: (focusOptions?: FocusOptions | undefined) => void;
+        focusLast: (focusOptions?: FocusOptions | undefined) => void;
+        observeUsing(observer: IntersectionObserver | ResizeObserver): void;
+        unobserveUsing(observer: IntersectionObserver | ResizeObserver): void;
+        getClientRects(): Array<DOMRect>;
+        getRootNode(getRootNodeOptions?: GetRootNodeOptions | undefined): Document | ShadowRoot | FragmentInstance;
+        addEventListener(
+            type: string,
+            listener: EventListener,
+            optionsOrUseCapture?: Parameters<Element["addEventListener"]>[2],
+        ): void;
+        removeEventListener(
+            type: string,
+            listener: EventListener,
+            optionsOrUseCapture?: Parameters<Element["removeEventListener"]>[2] | undefined,
```

Comment:
Why does this need `| undefined` but not the `addEventListener` one? Shouldn't it just be whatever `Element["removeEventListener"]` said?

Reply:
Just leftover from before I used `Parameters<...>`.